### PR TITLE
fix: data-form-fill

### PR DESCRIPTION
**Pull request by Prerna290**
## Type
- (X) Fix
- ( ) Refactor
- ( ) Style
- ( ) Docs
- ( ) Feat
- ( ) Hotfix

## Screenshots
image
image
image
image
image
image
image

## Bundle Size
4.87 MB

### CODE REVIEW

**Comment by arpansac** [comment]
Please check the card gap @Prerna290

**Comment by arpansac** [comment]
Please add * on required questions in super script <sup>

File: apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .event-location-tracks {
-  max-width: 1500px;
+  // max-width: 1500px;
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.scss]: Please Remove if not required

File: apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.scss [hunk 2]
```diff
@@ -17,14 +17,17 @@
     }
 
     nb-accordion-item-header {
-      @apply com-py-1 com-px-2;
+      @apply com-py-1 com-px-2 com-overflow-x-auto com-justify-between;
       background-color: #ffaa00;
       @apply com-border-0 com-rounded-md;
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.scss]: let's make this as a single line and convert background color to tailwind css

File: apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.ts [hunk 3]
```diff
@@ -216,7 +220,7 @@ export class EventLocationTracksComponent implements OnInit, AfterViewInit {
       // @ts-ignore
       end_time: eTimeNew,
       session_title: trackSlot.session_title,
-      tags_list: trackSlot.tags_list,
+      // tags_list: trackSlot.tags_list,
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.ts]: remove this if not required

File: NX
Comment (ArshdeepGrover) [inline NX:1]: remove this

**Review by ArshdeepGrover — CHANGES_REQUESTED** [review]

File: apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.scss (hunk 1 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.scss]: Done

File: apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.scss (hunk 2 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.scss]: Done

File: apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.ts (hunk 3 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.ts]: Done

File: NX
Comment (Prerna290) [reply NX:1]: Done

**Review by ArshdeepGrover — APPROVED** [review]

**Pull request merged**